hotnews/index.html

```diff
@@ -1,37 +1,107 @@
 <html>
-  <head>
-    <title>HotNews.ro</title>
-    <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicon.png">
 
-    <script src="https://cdn.tailwindcss.com"></script>
-    <script>
-      tailwind.config = {
-        theme: {
-          extend: {
-            colors: {
-              hotblue: '#2145ac',
-              hotgreen: '#3f9863',
-              hotred: '#eb3223'
-            }
+<head>
+  <title>HotNews.ro</title>
+  <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicon.png">
+  <link rel="stylesheet" href="hotnews.css">
+
+  <script src="https://cdn.tailwindcss.com"></script>
+  <script>
+    tailwind.config = {
+      theme: {
+        fontSize: {
+          font: '15px',
```

Comment:
Just to ensure you understand these configurations, by defining these 3 fonts you extend the tailwind font options with the following classes:

```
text-font (15px)
text-font2 (18px)
text-font3 (14px)
```

BUT
a) you're not using these classes, instead you preferred to use inline definitions (eg. text-[18px])
b) the naming can be better, eg
```
fontSize: {
  sm-1: '15px', // text-sm-1, because text-sm is 14px
  base-1: '18px' // text-base-1 because text-base is 16px and text-lg is 20px
}
```